exampleSite/content/docs/shortcodes.md

```diff
@@ -84,7 +84,7 @@ Include content from seperate file with syntax highlighting.
 
 ### file: Parameters
 
-0 => filename [`String`] \(required\)  
```

Comment:
Don't remove this spaces. Those are there for a reason. In markdown, double space at the end of line inserts a line-break.

Reply:
Sorry, it was default SublimeText behavior -- to remove trailing spaces at line end. 
Will update it shortly.